### ISSUE TITLE
Implementation Plan

### ISSUE THREAD

**Issue by CFuterman** [SamanthaDawsonBanks/MCOMP_Team_Project_2017-2018#85]

**Comment by CFuterman** [comment]
When writing this document, I will be in need of some functional, non-functional and technical requirements. I will be starting with the functional, but will be in need of some non functional and technical so that I can start writing this. Thank you in advance.

**Comment by SamanthaDawsonBanks** [comment]
yeah sounds good work on it on tuesday

**Comment by CFuterman** [comment]
I will need to carry the over another week over into milestone 3 because due to coursework that I've to complete.

**Comment by CFuterman** [comment]
This is taking a bit longer than expected based on the template that i have found. Should hopefully have my 2nd draft ready for the meeting on Thursday morning for the more technical members of the team to re-read

**Comment by CFuterman** [comment]
As and when I get to them, I will need some people to just fill out a small subsection of the project as it is outside of my knowledge. First up, I know @stevempope  is in charge of security. If you could write a small paragraph for each subsection that will be good. This is all from a template I found, so it says what you will need to write about if you need help. 

2.5 Security and Privacy
2.5.1 System Security Features – Steve 
[This subsection of the Project Implementation Plan provides an overview and discussion of the security features that must be addressed when it is implemented. It should include the determination of system sensitivity. Reference the applicable security guidance documents.]

2.5.2 Security Set Up During Implementation - Steve
[This subsection of the Project Implementation Plan addresses security issues specifically related to the implementation effort, if any. For example, if LAN servers or workstations will be installed at a site with sensitive data preloaded on non-removable hard disk drives, address how security would be provided for the data on these devices during shipping, transport, and installation because theft of the devices could compromise the sensitive data.]

**Comment by SamanthaDawsonBanks** [comment]
any chance you can upload the draft you have so far so that we can see the direction you are working in (it will help us to know what structure we need to write our sections in)

**Comment by CFuterman** [comment]
You should be able to see it in: OD/Documentation

**Comment by CFuterman** [comment]
Just as a reminder of the layout of how i would like everyone to input the major tasks: just a couple of issues each under whichever subtitle you feel you completed a major task. example of layout is below:

•	Motors and power supply fitted, wheels attached - #56
The motors needed to be wired to their shields. The power supply are batteries and the wires had to be sorted out. We came to a problem when we saw the thickness of the wires that came with the power supply did not fit in the motor shield. We have come up with an alternative power supply. When it came to the wheels, this took the longest part of all as we weren’t sure what size wheels to implement; so, they were the last component of the robot to be ordered and to be implemented to the robot. 

•	Creating Drive method - #83
This includes coming up with the mathematics that make the robot move. The algorithm that we have come up with can only be implemented as and when the team know the size of the wheels, which in the above task (#56) was the most troubling part for the team.
